File: Trabalho-Logica-de-Programação/C/questao3.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int num_pessoas;

    printf("Digite o número de pessoas: ");
    scanf("%d", &num_pessoas);

    char nomes[num_pessoas][50];
    int idades[num_pessoas];

    for (int i = 0; i < num_pessoas; i++) {
        printf("Digite o nome da pessoa: ");
        scanf("%s", nomes[i]);

        printf("Digite a idade da pessoa: ");
        scanf("%d", &idades[i]);
    }

    int indice_mais_velho = 0;
    
    for (int i = 1; i < num_pessoas; i++) {
        if (idades[i] > idades[indice_mais_velho]) {
            indice_mais_velho = i;
        }
    }

    printf("A pessoa mais velha é: %s\n", nomes[indice_mais_velho]);

    return 0;
}

File: Trabalho-Logica-de-Programação/Python/questao3.py
nomes = []
idades = []

for i in range(1, 4):
    nome = input(f'Digite o nome da {i}ª pessoa: ')
    idade = int(input(f'Digite a idade da {i}ª pessoa: '))

maiorIdade = idades[0]
maisVelho = nomes[0]

for i in range(3):
    if(idades[i] > maiorIdade):
        maiorIdade = idades[i]
        maisVelho = nomes[i]

print(f'O mais velho {maisVelho} com {maiorIdade}')